telebot/types.py

```diff
@@ -6428,15 +6427,20 @@ def de_json(cls, json_string):
             obj['thumbnail'] = None
         return cls(**obj)
 
-    def __init__(self, name, title, sticker_type, is_animated, is_video, stickers, thumbnail=None, **kwargs):
+    def __init__(self, name, title, sticker_type, is_animated=None, is_video=None, stickers=None, thumbnail=None, **kwargs):
```

Comment:
hmm.. setting stickers to None. Hopefully this will not get us problems in future?
Why not change order of params, as the class represents the sent sticker;

Reply:
We cannot change the order of parameters, it will break code which passed is_animated/is_video, i.e. **all code** who used this class...

Reply:
It is JsonDeserializable and just represents an object sent by API
So it is not a breaking change.

Reply:
Looks you are right.